xarray/core/dataarray.py

```diff
@@ -3776,6 +3776,8 @@ def fill_gaps(
         """Fill in gaps in the data using one of several filling methods.
         Allows for fine control on how far to extend the valid data into the gaps and the maximum size of the gaps to fill.
 
+        *Requires bottleneck.*
```

Comment:
I think numbagg should work too? (And is preferred so should be listed first)

Reply:
True! Also changed for ffill/bfill docs